GoogleForCreators/web-stories-wp
# Dashboard: "My Stories" → "Dashboard"

## Context
Update requested for dashboard to change name of "My Stories"


## Summary
Updates all "My Stories" titles in the dashboard to "Dashboard"


## Relevant Technical Choices
NA


## To-do
NA


## User-facing changes
in both the wp nav as well as the in app side navigation, the main page of the dashboard should now be referred to as `Dashboard` instead of `My Stories`

<img width="1436" alt="Screen Shot 2021-08-20 at 3 40 29 PM" src="https://user-images.githubusercontent.com/35983235/130291140-5eb037db-bc19-4f88-83c5-0ce3cb23416f.png">



## Testing Instructions
Open the dashboard, see that there's no more `My Stories` instances and they should all be `Dashboard` now





This PR can be tested by following these steps:

1.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #5646
